app/javascript/crayons/Icons/__stories__/Icons.mdx

```diff
@@ -1,30 +1,56 @@
+<mark>Please disregard generated HTML output in one of the tabs in the panel below. This one is a special snowflake for which you should use rails helper instead.</mark>
```

Comment:
```suggestion
<mark>Please disregard generated HTML output in one of the tabs in the panel below. This one is a special snowflake for which you should use the below rails helper instead.</mark>
```

maybe just to be explicit there's guidance on the helper below?

Reply:
👍